src/redist/redist.csproj

```diff
@@ -137,6 +137,12 @@
       BuildInParallel="False"
       Projects="@(SdksToBundle)">
     </MSBuild>
+
+    <!-- The WPF SDK .nupkg includes some files in the root that we don't want to bundle, so delete them -->
+    <ItemGroup>
+      <SDKFilesToDelete Include="$(ToolsetOutputDirectory)/Sdks/Microsoft.NET.Sdk.WPF/*" />
```

Comment:
Why are the files in the nupkg to begin with? Didn't you create it?

Reply:
It's created using the corefx build system which automatically adds some files (such as a license).